### PR TITLE
Add contributor summit EU under events

### PR DESCRIPTION
**Pull request by alisondy**
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: events/2022/05-contributor-summit/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+approvers:
+- sig-contributor-experience-leads # Replace when summit leads are decided
+
+labels:
+- area/contributor-summit
+- area/na-summit
```
Comment (mrbobbytables) [inline events/2022/05-contributor-summit/OWNERS:6]:
```suggestion
- area/eu-summit
```

File: events/2022/05-contributor-summit/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## What
+
+The Kubernetes Contributor Summit brings together new and current Kubernetes contributors to connect and learn from one another. 
+It is an opportunity for existing contributors to help shape the future of the project, and offers a welcoming space for new community members to learn, explore and put the contributor workflow to practice.
+The summit spans two days - an optional social event in the evening of Sunday, May 15th along with the main full-day event on Monday, May 16th.
+
+## Registration
+
+- [Sign the CLA](/CLA.md) if you have not done so already.
```
Comment (mrbobbytables) [inline events/2022/05-contributor-summit/README.md:11]:
```suggestion
```
They have to open a PR to sign now - no preemptive sign up

File: events/2022/05-contributor-summit/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# 2022 Kubernetes Contributor Summit EU
+
+## What
+
+The Kubernetes Contributor Summit brings together new and current Kubernetes contributors to connect and learn from one another. 
+It is an opportunity for existing contributors to help shape the future of the project, and offers a welcoming space for new community members to learn, explore and put the contributor workflow to practice.
+The summit spans two days - an optional social event in the evening of Sunday, May 15th along with the main full-day event on Monday, May 16th.
```
Comment (mrbobbytables) [inline events/2022/05-contributor-summit/README.md:7]: IIRC, everything is on the 16th

File: events/2022/05-contributor-summit/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## When and Where
+
+The Contributor Summit takes place in the days leading up to [KubeCon + CloudNativeCon EU](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/), make sure you plan travel accordingly.
+
+- Sunday, May 15th, 2022
+  Social event (optional)
+- Monday, May 16th, 2022
+  Full day event
```
Comment (mrbobbytables) [inline events/2022/05-contributor-summit/README.md:22]: similar to the above, IIRC its just monday

**Review by mrbobbytables — COMMENTED** [review]
few small things before kicking this off \o/

File: events/2022/05-contributor-summit/OWNERS (hunk 1 above)
Comment (alisondy) [reply events/2022/05-contributor-summit/OWNERS:6]: Good catch!

File: events/2022/05-contributor-summit/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  Evening Reception
+- Valencia, Spain
+
+## Team
+
+| Role | Lead | Shadow | Notes |
+|---|---|---|---|
+| Event Lead | Alison Dowdney [@alisondy](https://github.com/alisondy) | | |
```
Comment (parispittman) [inline events/2022/05-contributor-summit/README.md:26]: do you know what other roles you'll need yet?

**Pull request closed without merging**